force-app/main/SDO_SCOM_TNC_ACKNOWLEDGEMENT/lwc/sdo_scom_tnc_acknowledgement_place_order/sdo_scom_tnc_acknowledgement_place_order.html

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div if:true={displayPlaceOrder} class="container">
+    <div class="container">
         <div class="input-container">
             <input
```

Comment:
You can use lighting input: https://developer.salesforce.com/docs/component-library/bundle/lightning-input/example